com.unity.render-pipelines.universal/Runtime/2D/Passes/Render2DLightingPass.cs

```diff
@@ -318,6 +318,8 @@ public override void Execute(ScriptableRenderContext context, ref RenderingData
 
             if (renderingData.cameraData.camera.cameraType == CameraType.Preview)
                 isLitView = false;
+
+            Light2DManager.InitializeFrame();
```

Comment:
Updating it every frame during runtime is not necessary. The conditions to update are....
Editor & Not Playing...